src/dataflow/components/nodes/controls/relay-select-control.tsx

```diff
@@ -57,15 +57,15 @@ export class RelaySelectControl extends Rete.Control {
         channelsForType.forEach( c => { if (c.type === ch.type && ch.hubId === c.hubId) count++; } );
         return `${ch.hubName}:${ch.type}${ch.plug > 0 && count > 1 ? `(plug ${ch.plug})` : ""}`;
       };
-
+      const titleClass = getChannelString(selectedChannel).includes("Select") ? "label unselected" : "label";
```

Comment:
Seems brittle to have this depend on the string. Almost guarantees that they'll want to change it to "Choose" soon. 😉 Is there an ID of some sort that could be used instead?

Reply:
We could at least easily pull the value out into a constant so there's only one source of truth for what that string is.